doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.md

```diff
@@ -88,7 +70,74 @@ $ kubectl logs raycluster-autoscaler-head-xxxxx -c autoscaler | tail -n 20
+
+For an example, check out the [config file](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.autoscaler.yaml) that we used in the above quickstart guide.
+
+### Upscaling and downscaling speed
+
+If needed, you can also control the rate at which nodes should be added to or removed from the cluster. For applications with many short-lived tasks, you may wish to adjust the upscaling and downscaling speed to be more conservative.
+
+Use the `RayCluster` CR's `autoscalerOptions` field to do so. The `autoscalerOptions` field
+carries the following subfields:
+
+**`upscalingMode`**: This controls the rate of Ray pod upscaling. The valid values are:
+    - `Conservative`: Upscaling is rate-limited; the number of pending worker pods is at most the number
+      of worker pods connected to the Ray cluster.
+    - `Default`: Upscaling is not rate-limited.
+    - `Aggressive`: An alias for Default; upscaling is not rate-limited.
+
+**`idleTimeoutSeconds`** (default 60s): This is the number of seconds to wait before scaling down an idle worker pod. Worker nodes are considered idle when they hold no active tasks, actors, or referenced objects (either in-memory or spilled to disk).
+
+### Configuring the autoscaler sidecar container
+
+The `autoscalerOptions` also provides some options for configuring the autoscaler itself. Usually, it should not be necessary to modify these options.
+
+**`resources`**: The `resources` subfield of `autoscalerOptions` sets optional resource overrides
+for the autoscaler sidecar container. These overrides
+should be specified in the standard [container resource
+spec format](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources).
+The default values are as indicated below:
+```
+resources:
+  limits:
+    cpu: "500m"
+    memory: "512Mi"
+  requests:
+    cpu: "500m"
+    memory: "512Mi"
+```
+
+The following `autoscalerOptions` suboptions are also available for testing and development of the autoscaler itself.
+
+**`image`**: This field overrides the autoscaler container image.
+If your `RayCluster`'s `spec.RayVersion` is at least `2.0.0`, the autoscaler will default to using
+**the same image** as the Ray container. (Ray autoscaler code is bundled with the rest of Ray.)
+For older Ray versions, the autoscaler will default to the image `rayproject/ray:2.0.0`.
+
+**`imagePullPolicy`**: This field overrides the autoscaler container's
+image pull policy. The default is `Always`.
+
+**`env`** and **`envFrom`**: These fields specify autoscaler container
+environment variables. These fields should be formatted following the
+[Kuberentes API](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables)
+for container environment variables.
+
+
+## Understanding the Ray Autoscaler in the Context of Kubernetes
```

Comment:
@DmitriGekhtman, I still believe there is a way to make this following section more actionable instead of discussion-based. But I left it here for now, and we can leave it for a follow-up PR. I don't think it is critical for a first version of the clusters docs.

Reply:
I agree!

For example, 
"don't try to use the HPA!"
"do use the k8s autoscaler!"

The main operational part to extract from the architecture diagram is
"FYI the autoscaler communicates with K8s API server using these RBAC permissions! Make sure that's allowed in your K8s cluster".

Reply:
Adding a note-to-self here:
https://github.com/ray-project/ray/issues/27803